src/select/multiselect/OptionRenderer.js

```diff
@@ -3,29 +3,42 @@ import PropTypes from 'prop-types'
 
 const OptionRenderer = ({
   getOptionLabel,
-  option,
+  hideCheckboxes,
   labelKey,
+  option,
   selected
 }) => {
+  const optionLabel = getOptionLabel({ option, labelKey, selected })
+
+  if (hideCheckboxes) {
+    return (
+      <span aria-selected={selected}>
```

Comment:
@sthomas1618 @jaymi-johnson I wonder if we should have a visual indicator of which items are selected if checkboxes are hidden.  In this example, if you have more options selected than we show, you can't tell what those selected options are 
![image](https://user-images.githubusercontent.com/32145363/121100399-9bf9ac00-c7ae-11eb-82e6-ad2cf80c21b3.png)

Reply:
I didn't want to customize this as much as much as we will in our student-app because it's very specific (see image). However, I could change the font color ONLY (not sure if we want to use a specific one or use one of the colors we already use). I also will add a className so it's easier for other people to target in order to customize it themselves.

<img width="343" alt="Screen Shot 2021-06-01 at 3 08 01 PM" src="https://user-images.githubusercontent.com/63797536/121186185-cf7f1980-c834-11eb-895e-8eeefafb219c.png">

Reply:
I think as long as we have a className and add an example to this story that'll be good enough.

Reply:
I added the className - looking into if I can target it within a story..